docs/resources/guardian.md

```diff
@@ -19,10 +19,11 @@ resource "auth0_guardian" "default" {
     provider      = "auth0"
```

Comment:
Not part of your code but might just clean it up: I'm unable to comment on the specific line, but this file has a typo at line 5 which could be adressed: _With this __reasource__, you can configure some of the MFA options_ -> _With this __resource__, you can configure some of the MFA options_